### ISSUE TITLE
How do you delete cookies/saved data for an individual site? 

### ISSUE THREAD

**Issue by pressRtowin** [cylonid/NativeAlphaForAndroid#101]
I think I searched just about the entire internet before submitting this issue, and somehow I couldn't find a single relevant answer anywhere. 

To be clear, I'm _not_ looking to run multiple sandboxed sessions or anything like that. I'd simply like to perform the _basic web browser functionality_ of clearing a site's saved data.

**Comment by cylonid** [comment]
You are right, clearing a single container's data is not possible in Native Alpha at the moment.

I will consider this for one of the next two releases.

**Comment by JdotCarver** [comment]
+1 for this feature.
The sole reason I downloaded NativeAlpha was to : 
* Have Progressive Web Apps in sandboxed containers
* Wipe the sandbox upon container close.

**Comment by JdotCarver** [comment]
### Current Workaround :
1. Enable sandboxing for your desired web app. 
2. When you're done using it and wish to clear cookies, go into the specific web app settings. (swipe from Left → Right).
3. Under the Sandbox Section, Disable and immediately Re-enable the "Enable Sandbox" toggle.

<img src="https://github.com/user-attachments/assets/11d5ca8a-acdf-4566-a4f9-099e9236ed07" width="360" height="336">

This re-created the sandbox, effectively wiping cookies. :)

_Hope this helps._  
🙏
